### ISSUE TITLE
Component: Button a11y on "icon only" button

### ISSUE THREAD

**Issue by teshi24** [primefaces/primeng#17002]
### Describe the bug

On a plain icon button without a label, when adding the title and ariaLabel attribute, I would expect the WAVE extension to report no error.
However, it reports an error, while also listing the aria-label as feature:
![image](https://github.com/user-attachments/assets/f215c72b-964c-4923-91e4-b3e6c08e0e03)
![image](https://github.com/user-attachments/assets/42d8af84-5118-40ff-9632-77e42503942f)

Probably, the accessibility setup of primeNg for the button should be checked.

### Environment

Brave browser, Linux & Windows

### Reproducer

https://stackblitz.com/edit/github-jjzt417f?file=src%2Fapp%2Fapp.component.html

### Angular version

18.2.11

### PrimeNG version

17.18.11

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

21.7.3

### Steps to reproduce the behavior

1. create example locally (with sandbox, wave doesn't show the same behaviour like locally somehow)
2. open wave browser extension to check accessibility issues (if not yet installed: https://wave.webaim.org/extension/)

### Expected behavior

- Area label shown as feature
- no a11y bug

**Comment by teshi24** [comment]
NVM, I the ariaLabel was not taken correctly, fixed it now
